src/node.js

```diff
@@ -21,9 +21,9 @@ export default class NodeSourceMap extends SourceMap {
     }
     this.sourceMapInstance.addVLQMap(
       mappings,
-      sources,
-      sourcesContent.map((content) => (content ? content : '')),
-      names,
+      JSON.stringify(sources),
+      JSON.stringify(sourcesContent.map((content) => (content ? content : ''))),
+      JSON.stringify(names),
```

Comment:
😳

Reply:
No fan of it either but apparently it’s faster